Dockerfile

```diff
@@ -13,7 +13,9 @@ LABEL com.github.actions.color="gray-dark"
 RUN apk --update add openjdk8-jre
 RUN apk add --no-cache git
 
-RUN npm install -g firebase-tools@9.9.0
+RUN npm -g update
+RUN npm i -g npm@latest
+RUN npm i -g firebase-tools
```

Comment:
This looks good! The only thing is that I specifically set the version because I was requested to change this to a docker image and updating the version forces a new docker image to be built. (It used to use this and rebuild every time the action was used but users didn't like how long it took).

Reply:
@w9jds Ah right, that makes sense. Give me a moment. I'll update my PR.